src/components/atoms/ImageInput/ImageInput.tsx

```diff
@@ -93,33 +95,33 @@ const ImageInput: React.FC<ImageInputProps> = ({
 
   return (
     <>
-      <label className={labelClasses}>
-        <input
-          accept={ACCEPTED_IMAGE_TYPES}
-          hidden
-          id={name}
-          onChange={handleFileInputChangeWrapper}
-          type="file"
-          ref={inputFileRef}
-        />
-        {loading && <ImageOverlay disabled>processing...</ImageOverlay>}
-
-        <span
-          className={classNames("ImageInput__upload-button", {
-            "ImageInput__upload-button--small": small,
-            "ImageInput__upload-button--hidden": !!imageUrl,
-          })}
-        >
-          Upload
-        </span>
-      </label>
-
-      <input type="hidden" name={fileUrl} ref={register} readOnly />
-      {isInputHidden && (
-        <ButtonNG onClick={onButtonClick} variant="primary">
-          {text}
-        </ButtonNG>
+      {!!imageUrl && (
+        <label className={labelClasses}>
+          <input
+            accept={ACCEPTED_IMAGE_TYPES}
+            hidden
+            id={name}
+            onChange={handleFileInputChangeWrapper}
+            type="file"
+            ref={inputFileRef}
+          />
+          {loading && <ImageOverlay disabled>processing...</ImageOverlay>}
+
+          <span
+            className={classNames("ImageInput__upload-button", {
+              "ImageInput__upload-button--small": small,
+              "ImageInput__upload-button--hidden": !!imageUrl,
+            })}
+          >
+            Upload
+          </span>
+        </label>
       )}
+      <input type="hidden" name={fileUrl} ref={register} readOnly />
+      <div className="ImageInput__upload-container">
+        <ButtonNG onClick={onButtonClick}>{text}</ButtonNG>
+        <div className="ImageInput__upload-subtext">{subtext}</div>
+      </div>
```

Comment:
Don't know if adding `upload` in all names is better than just not adding it to any in this case.

Reply:
thanks, done b18441c